dluhc-component-library/src/components/timetablePage/columnDefinitions.tsx

```diff
@@ -10,11 +10,11 @@ export const columnDefinitions = [
     header: () => <span>Stage</span>,
   }),
   columnHelper.accessor("startDate", {
-    cell: (data) => data.getValue(),
+    cell: (data) => new Date(data.getValue()).toDateString(),
     header: () => <span>Start Date</span>,
   }),
   columnHelper.accessor("endDate", {
-    cell: (data) => data.getValue(),
+    cell: (data) => new Date(data.getValue()).toDateString(),
```

Comment:
For these two dates in this file could we limit it to just showing Month and Year. I'm just thinking with the form the input for these is only month and year:
![image](https://github.com/digital-land/plan-making/assets/47323438/f22d6d99-5d49-46b3-8d61-a18fa10d7c80)